internal/proxyattributes/proxyattributes_test.go

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
optional: I haven't completed the entire review. If you can test the setting of proxy attributes in an e2e style test, you can get rid of these tests.